JitsiConference.js

```diff
@@ -142,10 +148,11 @@ function JitsiConference(options) {
 
     /**
      * Flag set to <tt>true</tt> when P2P session has been established
-     * (ICE has been connected).
+     * (ICE has been connected) and this conference is currently in the peer to
+     * peer mode (P2P connection is the active one).
      * @type {boolean}
      */
-    this.p2pEstablished = false;
+    this.p2p = false;
```

Comment:
So the only purpose of this field is to say "now we're using P2P", but that can happen only once the ICE for the P2P connection has been established (reached "connected" for the first time). It doesn't work in the opposite direction tough - when it's true there's no guarantee that the P2P ICE is currently in "connected" state. It can be in "disconnected" for a moment when there is some network disruption. We won't switch back to JVB until P2P ice goes to FAILED. This flag should never be confused with ICE state.